docs/_rpc/ns-debug.md

```diff
@@ -88,9 +113,33 @@ storage and code).
 }
 ```
 
+### debug_freeOSMemory
+
+Forces garbage collection
+
+| Client  | Method invocation                                 |
+|:--------|---------------------------------------------------|
+| Go      | `debug.freeOSMemory()`                            |
+| Console | `debug.freeOSMemory()`                            |
+| RPC     | `{"method": "debug_freeOSMemory", "params": []}`  |
+
+
+
+### debug_freezeClient
+
+Forces a temporary client freeze, normally when the server is overloaded.
+Available as part of LES light server.
+
+| Client  | Method invocation                                    |
+|:--------|------------------------------------------------------|
+| Console | `debug.freezeClient(node string)`                    |
+| RPC     | `{"method": "debug_freezeClient", "params": [node]}` |
+
```

Comment:
Does this really exist @zsfelfoldi 
```
> debug.freezeClient()
Error: Invalid number of input parameters to RPC method
	at web3.js:5026:46(12)
	at web3.js:5069:22(27)
	at send (web3.js:5093:39(14))
	at <eval>:1:19(3)

> debug.freezeClient("a")
WARN [05-06|19:58:12.180] Served debug_freezeClient                reqid=16 duration="27.689µs"  err="the method debug_freezeClient does not exist/is not available"
Error: the method debug_freezeClient does not exist/is not available
	at web3.js:6365:37(47)
	at send (web3.js:5099:62(35))
	at <eval>:1:19(4)
```

Reply:
Running geth with `--light.serve`, then:

```
> debug.freezeClient("a")
Error: missing 'enr:' prefix for base64-encoded record
```

Reply:
Also kinda funny hack:

I had `--maxpeers 0 --light.serve 25` geth complained:

```
invalid peer config: light peer count (100) >= total peer count (100)
```

Then `--maxpeers 0 --light.serve 25 --light.maxpeers 0`, still geth complained:

```
invalid peer config: light peer count (0) >= total peer count (0)
```

Finally the hack, `--maxpeers 0 --light.serve 25 --light.maxpeers -1`:

```
success
```

😏